charts/DataTableTransform.ts

```diff
@@ -148,17 +149,28 @@ export class DataTableTransform {
         return TargetYearMode.point
     }
 
+    @computed get minYear(): Time {
+        return this.chart.data.minYear ?? 1990
```

Comment:
nit: any reason why the defaults for this are 1990 and 2020, and 1900 and 2000 for the others? Maybe setting them as a constant like ARBITRARY_MIN_YEAR might be clearer.